crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/builder/AppealContributionResponseBuilder.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.laa.crime.contribution.builder;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.justice.laa.crime.contribution.model.AppealContributionResponse;
+import uk.gov.justice.laa.crime.contribution.model.Contribution;
+import uk.gov.justice.laa.crime.contribution.staticdata.enums.TransferStatus;
+
+
+@Component
+@AllArgsConstructor
+public class AppealContributionResponseBuilder {
```

Comment:
Technically speaking this class `AppealContributionResponseBuilder` is not a builder (it doesn't follow the Builder pattern) although I can see it does use builder pattern defined in another class `AppealContributionResponse`.

It's a bit personal personal preference, I would tend to name classes like this mappers i.e. `AppealContributionResponseMapper` 

Se below for details of the builder pattern in Java
https://www.digitalocean.com/community/tutorials/builder-design-pattern-in-java